lib/engine.js

```diff
@@ -305,6 +305,14 @@ module.exports = function (s, conf) {
     })
   }
 
+  function isOrderTooSmall(product, quantity, price) {
+    if (product.min_size && Number(quantity) < Number(product.min_size))
```

Comment:
There is no need, since `product.min_size` will resolve in `undefined` if it does not exists (and thus the condition will fails).
The only possible crash would be if `product` was `null` or `undefined`